docs/docs/dev-server/overview.md

```diff
@@ -7,7 +7,7 @@ eleventyNavigation:
   order: 1
 ---
 
-Dev Server for web applications, ideal for buildless es module workflows. Optionally supports simple code transformations.
+Web dev server helps developing for the web, using native browser features like es modules. It is ideal for buildless workflows, and has a plugin architecture for light code transformations.
```

Comment:
Do we want to use/establish a "how we talk about web dev server" standard? Is is `Web dev server`, `Web Dev Server`, `@web/dev-server`, etc...this will help the people who wright State of JS 2021.

Reply:
I think Web Dev Server for regular sentences, and `@web/dev-server` when refering to the package.

Reply:
I've updated all the references now